src/test/java/com/client/BaseTest.java

```diff
@@ -1,30 +1,34 @@
 package com.client;
 
-import static org.junit.Assert.assertNotNull;
-
 import java.util.List;
 
+import com.client.config.GeneralConfig;
+import com.client.config.TestConfig;
+import com.client.core.scheduledtasks.service.impl.StandardEventWorkflowFactory;
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.context.ApplicationContext;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit4.AbstractTransactionalJUnit4SpringContextTests;
-import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 
 import com.bullhornsdk.data.api.BullhornData;
 import com.bullhornsdk.data.api.mock.MockBullhornData;
-import com.client.core.ApplicationSettings;
 import com.client.core.base.tools.test.TestEntities;
 import com.client.core.base.tools.test.TestUtil;
 
-@RunWith(SpringJUnit4ClassRunner.class)
-@ContextConfiguration(locations = { "/applicationContext.xml", "classpath*:/*-workflow.xml", "classpath*:/*-config.xml",
-		"classpath*:/*-scheduledtasks.xml", "classpath*:/*-applicationContext.xml", "/*-test.xml" })
+
+@ExtendWith(SpringExtension.class)
+@ConfigurationPropertiesScan
+@ContextConfiguration(classes = { GeneralConfig.class, TestConfig.class, StandardEventWorkflowFactory.class})
```

Comment:
I could not figure out any other way to accomplish this portion.

The original code simply configures the context to look at '/applicationContext.xml' which would scan the `com.client.core` package for beans? But if I include this, then I'm getting errors about the StandardEmailTemplateService not having the org.thymeleaf bean